src/common/test/SocketTest.cpp

```diff
@@ -2,25 +2,75 @@
 
 #include <common/system/Socket.h>
 #include <common/testutils/TestBase.h>
+#include <stdio.h>
 
 std::vector<std::string> ipOk = {"0.0.0.0", "10.10.10.10", "127.0.0.1",
                                  "224.1.2.3", "255.255.255.255"};
-std::vector<std::string> ipNotOk = {"a.0.0.0", "1.2.3",       "1.2",
-                                    "",        "127.0.0.256", "metrics"};
+std::vector<std::string> ipNotOk = {"a.0.0.0", "1.2.3", "1.2",
+                                    "", "127.0.0.256", "metrics"};
+
+class SocketTest : public ::testing::Test
+{
+
+private:
+  void error(const char *msg)
```

Comment:
@JennyLouise 

Maybe if you would know a better way how to write out errors occurred during the test would be a nice critics. I'm not sure this is a best approach.